docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc

```diff
@@ -23,27 +19,29 @@ For Maven:
 </dependency>
 ----
 
-`quarkus-opentelemetry` extension already includes the default OTLP exporter. To deactivate that default exporter and only have this Azure extension exporting data, please set the following property in the `application.properties`:
-
-[source]
-----
-quarkus.opentelemetry.tracer.exporter.otlp.enabled=false
-----
-
-Enable the Azure export by adding following properties in the `application.properties`:
+Then enable the Azure export:
 
 [source]
 ----
 quarkus.opentelemetry.tracer.exporter.azure.enabled=true
 ----
 
-You need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string [in the top right corner](https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string).
+You also need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string [in the top right corner](https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string).
+
+Provide this connection string in the `application.properties` file:
 
 [source]
 ----
 quarkus.otel.azure.applicationinsights.connection.string=your_connection_string
```

Comment:
Is it supposed to work with an environmental variable?QUARKUS_OTEL_AZURE_APPLICATIONINSIGHTS.CONNECTION_STRING?